src/lTerm.ml

```diff
@@ -112,12 +112,12 @@ let () =
   match LTerm_unix.sigwinch with
     | None ->
         (* Check for size when something happen. *)
-        ignore (Lwt_sequence.add_r send_resize Lwt_main.enter_iter_hooks)
+        ignore (LTerm_sequence.add_r send_resize Lwt_main.enter_iter_hooks)
     | Some signum ->
         try
           ignore (Lwt_unix.on_signal signum (fun _ -> send_resize ()))
         with Not_found ->
-          ignore (Lwt_sequence.add_r send_resize Lwt_main.enter_iter_hooks)
+          ignore (LTerm_sequence.add_r send_resize Lwt_main.enter_iter_hooks)
```

Comment:
Not sure if this  will compile. Is ```'a LTerm_sequence.t = 'a Lwt_sequence.t```? If so, how may I ask?

Reply:
Checkout the lwt3.mli. We are making sure `'a LTerm_sequence.t = Lwt_sequence.t`